config/deployments/local_devnet/.sidecar.env

```diff
@@ -1,11 +1,8 @@
 NETWORK_ID=13527
-
-HOST=localhost
 L1_CHAIN_ID=1337
-L1_WS_PORT=8545
+L1_ENDPOINT=ws://172.17.0.1:8545
```

Comment:
use `127.0.0.1`?

Reply:
iirc localhost won’t work unless we set `—network=‘host’` when starting the L1 docker which isn’t supported on Mac. that’s weht the docker0 address is used here

Reply:
these are just configs anyways, people can change it if they need to